media/src/project/project-map.tsx

```diff
@@ -76,6 +76,9 @@ export const ProjectMap: React.FC = () => {
     const mapRef = useRef<MapRef>(null);
     const geocoderContainerRef = useRef<HTMLDivElement>(null);
 
+    const [showSearchPopup, setShowSearchPopup] = useState<boolean>(false);
+    const [searchResult, setSearchResult] = useState<Results | null>(null);
```

Comment:
I finally have to learn how to use React Hooks 😂 I got into this a bit when looking into using this library called TanStack Tables with Mediathread - seems some libraries require you to use React Hooks. I think I get the basics, but I'll have to read their docs yet again to really understand what's going on here.